indexprojects.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Projects</title>
+ <link rel="preconnect" href="https://fonts.googleapis.com">
+ <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+ <link href="https://fonts.googleapis.com/css2?family=Montserrat+Alternates&display=swap" rel="stylesheet">
+    <meta name="description" content="Projects HTML file">
+    <meta name="author" content="Phil Allen">
+    <link rel="stylesheet" href="css/normalize.css">
+    <link rel="stylesheet" href="css/styles.css">
+    <link rel="icon" href="iimages/codeicon.png" sizes="192x192">
+    <link rel="shortcut icon" href="images/codeicon.png">
+    <link rel="apple-touch-icon" href="images/codeicon.png">
+    <!-- <script defer src="js/index.js"></script> -->
+</head>
+<body>
+    <h1>Projects page</h1>
+    <div class="container">
```

Comment:
that same comment about divs

Reply:
Not changed as I couldn't find anything suitable